ui/package: Bump the core-ui version to v0.22.2.1

ChengYanJin opened the pull request:
**Component**: UI 

<!-- E.g. 'salt', 'containers', 'kubernetes', 'build', 'tests'... -->

**Context**: 
Bump the core-ui version to fix the broken tooltip in the charts.

**Summary**:
To fix the issue of the instance name with two dots cause no data in the
tooltip.

**Acceptance criteria**: 


---

<!-- Declare one or more issues to close once this PR gets merged -->

Closes: #ISSUE_NUMBER

<!-- If you want to refer to an issue while not closing it, use:

See: #ISSUE_NUMBER

-->

ChengYanJin committed:
ui/package: Bump the core-ui version to v0.22.2.1

To fix the issue of the instance name with two dots cause no data in the
tooltip.

bert-e commented:
# Hello chengyanjin,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/2.11/bugfix/bump-core-ui-version-to-v0.22.2.1` with contents from `bugfix/bump-core-ui-version-to-v0.22.2.1`
and `development/2.11`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/2.11/bugfix/bump-core-ui-version-to-v0.22.2.1 origin/development/2.11
 $ git merge origin/bugfix/bump-core-ui-version-to-v0.22.2.1
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/2.11/bugfix/bump-core-ui-version-to-v0.22.2.1
```

ChengYanJin committed:
ui/nodes: Fix the bug where we don't have series for IOPS chart

Introduce a technical debt because we add the isPlaneInterfaceRequired
prop in MetricSymmetricalChart, however we should useSymetricalChartSeries
hook when the median and quantile chart is ready.

ChengYanJin commented:
/status

bert-e commented:
# Status





*Status report is not available.*

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/bump-core-ui-version-to-v0.22.2.1` into
`development/2.10`
* [w/2.11/bugfix/bump-core-ui-version-to-v0.22.2.1](https://github.com/scality/metalk8s/compare/development/2.11...w/2.11/bugfix/bump-core-ui-version-to-v0.22.2.1) will be merged into `development/2.11` 
* [w/123.0/bugfix/bump-core-ui-version-to-v0.22.2.1](https://github.com/scality/metalk8s/compare/development/123.0...w/123.0/bugfix/bump-core-ui-version-to-v0.22.2.1) will be merged into `development/123.0` 



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @alexandre-allard-scality

* @gdemonet

* @thomasdanan

* @chengyanjin

* @lucieleonard

* @teddyandrieux

* @jbwatenbergscality

* @nicolast

ChengYanJin commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/2.11/bugfix/bump-core-ui-version-to-v0.22.2.1` with contents from `bugfix/bump-core-ui-version-to-v0.22.2.1`
and `development/2.11`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/2.11/bugfix/bump-core-ui-version-to-v0.22.2.1 origin/development/2.11
 $ git merge origin/bugfix/bump-core-ui-version-to-v0.22.2.1
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/2.11/bugfix/bump-core-ui-version-to-v0.22.2.1
```

ChengYanJin commented:
/status

bert-e commented:
# Status





*Status report is not available.*

bert-e commented:
# History mismatch



Merge commit #c18cde9caa0eadd48088bf7a8083c250684ac916 on the integration branch
`w/123.0/bugfix/bump-core-ui-version-to-v0.22.2.1` is merging a branch which is neither the current
branch `bugfix/bump-core-ui-version-to-v0.22.2.1` nor the development branch
`development/123.0`.

It is likely due to a rebase of the branch `bugfix/bump-core-ui-version-to-v0.22.2.1` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

bert-e commented:
# History mismatch



Merge commit #c18cde9caa0eadd48088bf7a8083c250684ac916 on the integration branch
`w/123.0/bugfix/bump-core-ui-version-to-v0.22.2.1` is merging a branch which is neither the current
branch `bugfix/bump-core-ui-version-to-v0.22.2.1` nor the development branch
`development/123.0`.

It is likely due to a rebase of the branch `bugfix/bump-core-ui-version-to-v0.22.2.1` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

ChengYanJin commented:
/status

bert-e commented:
# Status





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @alexandre-allard-scality

* @gdemonet

* @thomasdanan

* @chengyanjin

* @lucieleonard

* @teddyandrieux

* @jbwatenbergscality

* @nicolast

ChengYanJin commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.10`

* :heavy_check_mark: `development/2.11`

* :heavy_check_mark: `development/123.0`



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.10`

* :heavy_check_mark: `development/2.11`

* :heavy_check_mark: `development/123.0`



The following branches have **NOT** changed:

* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



Please check the status of the associated issue None.

Goodbye chengyanjin.
